### PR TITLE
PMM-13486 Add new metrics for Mongodb 8

### PR DESCRIPTION
**Pull request by YashSartanpara1**
[PMM-13486](https://perconadev.atlassian.net/browse/PMM-13486)

[PMM-13486]: https://perconadev.atlassian.net/browse/PMM-13486?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dashboards/MongoDB/MongoDB_Oplog_Details.json [hunk 1]
```diff
@@ -388,10 +388,12 @@
       "targets": [
         {
           "datasource": "Metrics",
-          "expr": "avg by (service_name) (mongodb_mongod_metrics_repl_buffer_count{service_name=~\"$service_name\"})",
+          "editorMode": "code",
```
Comment (matejkubinec) [inline dashboards/MongoDB/MongoDB_Oplog_Details.json:391]: Is editorMode right here? Since it wasn't there before.

Comment (rnovikovP) [reply dashboards/MongoDB/MongoDB_Oplog_Details.json:391]: Will this work for v2 ? or better   keep the only formula change?

Comment (YashSartanpara1) [reply dashboards/MongoDB/MongoDB_Oplog_Details.json:391]: The file was exported from v2 only, and tested with v3 as well.
let me double check

Comment (YashSartanpara1) [reply dashboards/MongoDB/MongoDB_Oplog_Details.json:391]: @matejkubinec Yes, its fine as it was included while exporting the dashboard from v2.

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**